main/settings.py

```diff
@@ -42,6 +42,15 @@
     SENTRY_TRACES_SAMPLE_RATE=(float, 0.2),
     SENTRY_PROFILE_SAMPLE_RATE=(float, 0.2),
     # App Domain
+    # Vector Database
+    QDRANT_DB_HOST=str,
+    QDRANT_DB_PORT=int,
+    QDRANT_DB_COLLECTION_NAME=str,
+    # Embedding Model
+    EMBEDDING_MODEL=int,
+    LLM=int,
+    # Ollama url
+    OLLAMA_BASE_URL=str,
```

Comment:
This is optional?
```
    OLLAMA_BASE_URL=str,
```

Reply:
This is optional but required when OLLAMA LLM is chosen

Reply:
Let's make this optional and do an assert in the settings.py if OLLAMA is selected.

Let's do the same for other models as well.